Release 0.6.8

jakirkham opened the pull request:
Fixes https://github.com/conda-forge/libgpuarray-feedstock/issues/2

* Releases `pygpu` 0.6.8.
* Change the recipe to build Python/NumPy combinations (instead of being a metapackage).
* Re-render with `conda-smithy` 2.3.2 to regenerate the CI matrices.

Note: This will only work after PR ( https://github.com/conda-forge/libgpuarray-feedstock/pull/5 ) is merged and packages for `libgpuarray` 0.6.8 are available.

jakirkham committed:
Convert metapackage to a real Python package

As the Python components will now be included in `pygpu` instead of
`libgpuarray`, these sections are uncommented so this can build the
Python portions and actually package them.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
If you have a chance to look at this too, @abergeron, that would be appreciated. Would note this is going to fail until `libgpuarray` 0.6.8 is built and released.

abergeron reviewed (approved):
Looks good

jakirkham commented:
Thanks. Repushed to queue the CIs after `libgpuarray`'s package release, but no actual change.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.3.2

Now that this has become a proper Python package. Re-render to get
Python and NumPy CI matrices created.

jakirkham committed:
Release 0.6.8
